src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs

```diff
@@ -478,9 +478,11 @@ public void FileRegionsCache_ProperlyCaches()
                 Region populated = fileRegionsCache.PopulateTextRegionProperties(copy, uri, populateSnippet: true);
             }
 
+            int timeThreshold = 150;
```

Comment:
`const`?

Reply:
:v:

Reply:
Got it. Sorry about that. I need to find a new model for performance tests which are probably relative to measured performance of something else just run in the same environment.

Reply:
Side note: in the agent 'wild', this code path is taking closer to 200 ms. You have a fine idea, though, we could update the test to run a baseline operation (no caching), then repeat with caching. We should see a substantive difference that's roughly consistent in relative terms. Will look at that.

---
In reply to: [397266682](https://github.com/microsoft/sarif-sdk/pull/1822#discussion_r397266682) [](ancestors = 397266682)